src/common_tasks/corpus_candidate_set.py

```diff
@@ -23,6 +23,18 @@ def validate_corpus_items(corpus_items: List[Dict]):
 
     return corpus_items
 
+@task()
+def validate_candidate_items(corpus_items: List[Dict]):
+    min_item_count = 1
```

Comment:
we may wish to increase this to 3 if we expect a full slate.

Reply:
Those are tests to verify the minimum state of the topic candidates and causing the flow to fail if those conditions are not met. 
If we are to increase the `min_item_count` to 3, we would have to apply filters to not have those topics when the candidates are less than 3.  Is that what you want?

Reply:
@lfishhead I have merged the changes for now. If the candidates per topic min count 3 is important, lets discuss it and make the change in the next iteration